### ISSUE TITLE
auto-sklearn fails to stop after an hour, produces 15G over the night

### ISSUE THREAD

**Issue by zygmuntz** [automl/auto-sklearn#146]
I run the software as follows:

	automl = autosklearn.classification.AutoSklearnClassifier( time_left_for_this_task = 3600, per_run_time_limit = 600 )
	automl.fit( x_train, y_train, metric = 'auc_metric' )

It didn't stop after an hour, ran overnight.

It also produced 15G of output in /tmp (filling the space left on the disk), which I consider a bit much for a dataset smaller than 30MB.

BTW, how do I find out the performance of the best model, after Ctrl^C in ipython? I tried this:

	In [12]: print automl.score( x_test, y_test )
	The history saving thread hit an unexpected error (OperationalError('database or disk is full',)).History will not be written to the database.
	/usr/local/anaconda/lib/python2.7/site-packages/sklearn/utils/__init__.py:93: DeprecationWarning: Function transform is deprecated; Support to use estimators as feature selectors will be removed in version 0.19. Use SelectFromModel instead.
	warnings.warn(msg, category=DeprecationWarning)
	/usr/local/anaconda/lib/python2.7/site-packages/sklearn/utils/__init__.py:93: DeprecationWarning: Function transform is deprecated; Support to use estimators as feature selectors will be removed in version 0.19. Use SelectFromModel instead.
	warnings.warn(msg, category=DeprecationWarning)
	/usr/local/anaconda/lib/python2.7/site-packages/sklearn/utils/__init__.py:93: DeprecationWarning: Function transform is deprecated; Support to use estimators as feature selectors will be removed in version 0.19. Use SelectFromModel instead.
	warnings.warn(msg, category=DeprecationWarning)
	0.0575066263215

Is 0.05 the AUC? A random model would score 0.5.

**Comment by mfeurer** [comment]
This is indeed weird and should not happen, sorry. Can you provide some information on the dataset you used? The scores in auto-sklearn are a known problem since they are rescaled so that 0 is a random prediction. Thus, 0.05 would translate to 0.525, which is still pretty much random.

**Comment by zygmuntz** [comment]
I'm using the numer.ai dataset from June. AUC 0.525 is about what you get with logistic regression, which is a good benchmark for this set.

**Comment by mfeurer** [comment]
Could you please provide me the output log of auto-sklearn?

**Comment by mfeurer** [comment]
I just released version 0.1 on pypi. Could you please check whether this problem persists?

**Comment by zygmuntz** [comment]
Thank you, this time it stopped after an hour.
